python/TestHarness/util.py

```diff
@@ -67,6 +67,10 @@
                      'default'   : 'FALSE',
                      'options'   : {'TRUE' : '1', 'FALSE' : '0'}
                    },
+  'cxx11' :        { 're_option' : r'#define\s+LIBMESH_HAVE_CXX11\s+(\d+)',
+                     'default'   : 'FALSE',
+                     'options'   : {'TRUE' : '1', 'FALSE' : '0'}
+                   },
```

Comment:
@jwpeterson - Is this the best way to detect MOOSE with C++11 in the TestHarness? It looks ok, just wanted to make sure you think so too.